spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/TracingProperties.java

```diff
@@ -194,4 +203,26 @@ enum PropagationType {
 
 	}
 
+	public static class Brave {
+
+		/**
+		 * This signals to Brave (tracing library) that the propagation type and the
```

Comment:
Given that the property's prefix is `management.tracing.brave` do we need `(tracing library)` in the description? Put another way, is there another Brave that could cause confusion?

Reply:
We try to have `boolean` properties start with `Whether the…`. Perhaps this could be something like this instead:

> Whether the propagation type and tracing backend support sharing the span ID between client and server spans. Requires B3 propagation and a compatible backend.

Reply:
I only know about the browser and there is a video editor too. I don't think it is confusing without "(tracing library)" in the description especially you need to add the Micrometer Tracing Brave Bridge.